_deploy/r/demo/emission/emission.gno

```diff
@@ -0,0 +1,19 @@
+package emission
+
+import (
+	"std"
+
+	pusers "gno.land/p/demo/users"
+	"gno.land/r/demo/gnoswap/consts"
+	"gno.land/r/demo/gns"
+)
+
```

Comment:
// emissionAddr is the address from which GNS tokens will be emitted.

Reply:
92eedc4d4a3245e6a9376a75bac78d54e9a5f389